pkg/asset/quota/aws/aws.go

```diff
@@ -133,11 +135,19 @@ func machineTypeToQuota(t string, instanceTypes map[string]InstanceTypeInfo) quo
 	if !ok {
 		return quota.Constraint{Name: "ec2/L-7295265B", Count: 0}
```

Comment:
Can we add a warning to the places where we are returning a zero-count quota so that the user is informed that we are not actually checking the quota? That will help us know in the future when users start trying to use new instance classes.